fetch_gazebo/CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package fetch_gazebo
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.7.0 (2015-10-21)
```

Comment:
please revert all changes to this file -- this is done as part of the debian release